### PR TITLE
add brackets.scm

### PR DESCRIPTION
**Pull request by kylepritchard**
Add highlighting of matching brackets. So used to it from other editors and would love it in V for zed

### CODE REVIEW

File: languages/v/brackets.scm [hunk 1]
```diff
@@ -0,0 +1,7 @@
+("(" @open ")" @close)
+("[" @open "]" @close)
+("{" @open "}" @close)
+("\"" @open "\"" @close)
+("'" @open "'" @close)
+("<" @open ">" @close)
+("`" @open "`" @close)
```
Comment (lv37) [inline languages/v/brackets.scm:7]:
```suggestion
```
Quotes don't need to be highlighted since they don't ever get nested

Comment (lv37) [inline languages/v/brackets.scm:7]:
```suggestion
("`" @open "`" @close)
(closure_parameters "|" @open "|" @close)
```

**Comment by kylepritchard** [comment]
I agree with the back ticks, the quotes could be from multiline strings but they are easy to track.

Looking at the other langs and option for interpolation could be added?

(interpolation "#{" @open "}" @close)

On my phone now so can't test properly or the effect (new to zed)

File: languages/v/brackets.scm [hunk 2]
```diff
@@ -0,0 +1,5 @@
+("(" @open ")" @close)
+("[" @open "]" @close)
+("{" @open "}" @close)
+
+(interpolation "${" @open "}" @close)
```
Comment (lv37) [inline languages/v/brackets.scm:5]:
```suggestion
(string_interpolation (interpolation_opening) @open (interpolation_closing) @close)
```

Comment (kylepritchard) [reply languages/v/brackets.scm]: Read the grammar.js file and see where the query comes from now

**Comment by lv37** [comment]
Thanks

**Pull request merged**